Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
```

Comment:
### [better_errors](https://github.com/BetterErrors/better_errors)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.5.1 → 2.7.1

[_[change-log](https://github.com/BetterErrors/better_errors/releases), [source-code](https://github.com/BetterErrors/better_errors)_]





<details>
<summary>Commits</summary>

A change of **39** commits. See the full changes on [the compare page](https://github.com/BetterErrors/better_errors/compare/v2.5.1...v2.7.1).

These are the first 10 commits:
- (679b7de) [Fix NoMethodError when variables cannot be gotten](https://github.com/BetterErrors/better_errors/commit/679b7deca53e50c22c14a103cd3f4972e459622d)
- (de57472) [Specify older kramdown on older ruby versions](https://github.com/BetterErrors/better_errors/commit/de57472ae9bda88add83209e1d4842857c12266b)
- (a331158) [Fix i18n requiring Ruby 2.3+](https://github.com/BetterErrors/better_errors/commit/a331158786a4896d93bd228290ea32aa7490da35)
- (34df3eb) [Merge pull request #437 from BetterErrors/fix/kramdown-ruby-…](https://github.com/BetterErrors/better_errors/commit/34df3eb37d94f114410642d62b34270741b391b4)
- (1891ad8) [Merge pull request #430 from RobinDaugherty/fix/nomethoderro…](https://github.com/BetterErrors/better_errors/commit/1891ad8c227ba67279ecdb2e8c4c0aca35b2726e)
- (361d506) [Allow passing IPAddrs to allow_ip](https://github.com/BetterErrors/better_errors/commit/361d5065cad815234bc44b9c5601f6479223a5f0)
- (5e1d687) [Merge pull request #444 from jdelStrother/allow-ipaddr](https://github.com/BetterErrors/better_errors/commit/5e1d687e3cabcc39dcfe8be2e729b4eb4ec5e7cb)
- (faa39ec) [Allow skipping variable inspection by class name](https://github.com/BetterErrors/better_errors/commit/faa39ec4a2f8233a194a33bbd383a7c302f25dc1)
- (da5e655) [Fix Travis CI Build Problems](https://github.com/BetterErrors/better_errors/commit/da5e6556b13a30634912405c56b725a0ee20174c)
- (641da2a) [Fix spec errors with binding of caller](https://github.com/BetterErrors/better_errors/commit/641da2ac4d383a781dfc0acb23b9c7e5f17220f9)


</details>